contracts/interfaces/IOrderNotificationReceiver.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.11;
+
+import "../LimitOrderProtocol.sol";
+
+interface IOrderNotificationReceiver {
+    function notifyOrderBroadcasted(
+        LimitOrderProtocol.Order memory _order,
+        address _caller
```

Comment:
I think this one is redundant because the `_order` already has `maker`.

Is there a reason why we need to distinguish between `_caller` and `order.maker`?